<?xml version="1.0"?>
<launch>
  <arg name="namespace" default="" />
  <arg name="bag_name" default="1" />
  

  <node name="tello" pkg="dronet_tello" type="tello.py" output="screen"/>
  <node name="gtg_dyn" pkg="dronet_tello" type="gtg_dyn2.py" output="screen" args="$(arg x) $(arg y)"/>
  <node name="key_vel_middleman" pkg="dronet_tello" type="key_vel_middleman.py" output="screen"/>

  <!--<node name="tello_key_vel" pkg="dronet_tello" type="key_vel_toggle.py" /> -->
  <!--<node name="user_input" pkg="dronet_tello" type="user_input.py" output="screen"/> -->
  <node pkg="rosbag" type="record" name="rosbag_record" args="record -o $(find dronet_tello)/../../demo3_statestd /rosout /velocity /flight_data /state /user_input /obstacle_detector /keys_enabled /user_velocity /obstacle_dyn /vicon/TELLO/TELLO /vicon/OBSTACLE/OBSTACLE" />

</launch>
